products-service/handlers/get-product-by-id.ts

```diff
@@ -0,0 +1,36 @@
+import { APIGatewayProxyHandler } from 'aws-lambda';
+import * as products from '../products/products.json';
+import 'source-map-support/register';
+import { corsHeaders, errorMessages } from '../utils';
+
+export const getProductById: APIGatewayProxyHandler = async (event, _context) => {
+    try {
```

Comment:
I'd suggest moving duplicate logic (like error handling, response formatting) into separate lib/ functions and use them across the handlers.

It's a wish for the future, no need to implement it right now. It won't affect your score.

Reply:
Agreed